css/main.css

```diff
@@ -1,3 +1,861 @@
+        grid-gap: 50px;
+        grid-template-columns: repeat(3, 1fr);
+        grid-template-rows: repeat(1, 1fr); 
+        margin: 10px auto 0 30px;
+        padding: 0 30px;
+        max-width: 100%; */
+        display: flex;
+        flex-direction: row;
+        gap: 20px;
+        border: 2px dotted red;
+        height: 700px;
+        margin-left: 30px;
+    }
+
+    .section4-column-wrapper1 {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: flex-start;
+        text-align: left;
+        padding-right: 40px;
+        font-size: 1.06rem;
+        border: 2px dotted blue;
+        align-self: flex-end;
+    }
+
+    .section4-column-wrapper2 {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        justify-content: flex-start;
+        text-align: left;
+        padding-right: 40px;
+        font-size: 1.06rem;
+        border: 2px dotted blue;
+    }
+
+    .section4-img1 {
+        width: 310px;
+        height: 390px
+    }
+
+    .section4-img2 {
+        width: 310px;
+        height: 390px;
+        margin: 0;
+        padding: 0;
+    }
+
+    .section4-img3 {
+        width: 310px;
+        height: 390px;
+        margin: 0;
+        padding: 0;
+    }
+
+    .section4-sub-title {
+        padding: 0;
+        font-size: 2.1875rem;
+    }
+
+    .section4-paragraph {
+        padding-right: 40px;
+    }
+
+}
+
+/* Section 5 */
+.section5-wrapper {
+    padding: 20px;
+}
+.section5-img {
+    width: 280px;
+    height: 371px;
+    margin-bottom: 20px;
+}
+
+.section5-h2 {
+    margin-bottom: 0;
+}
+
+.section5-p {
+    margin-top: 0;
+} 
+
+.section5-button {
+    font-family: Raleway;
+    font-weight: 300;
+    padding: 12px 50px;
+    margin: 10px 0 20px 0;
+    border: 2px solid var(--black);
+    font-size: 1em;
+    background-color: var(--white);   
+}
+
+/* Section 6 Styles */
+
+.section6-content-wrapper {
+    background-color: var(--light-green);
+    padding: 40px 30px 15px 30px;
+}
+
+.section6-border {
+    border: 1px solid var(--black);
+    border-radius: 50%;
+    display: inline;
+    padding: 15px;
+}
+
+.section6-p {
+    margin-top: 10px;
+    margin-bottom: 80px;
+}
+
+/* Footer */
+
+.footer-wrapper {
+    padding: 20px 0 5px 0;
+}
+
+.footer-h2 {
+    margin: 15px 0 0 0;
+}
+
+.footer-main-link {
+    color: black;
+}
+
+.footer-sub {
+    background-color: var(--dark-green);
+    padding: 0 20px 0 20px;
+    color: var(--white);
+    margin-top: 190px;
+}
+
+.footer-overlap-imgs {
+    position: relative;
+    top: 0;
+    left: 0;
+}
+
+.footer-img1 {
+    z-index: 300;
+    width: 123px;
+    height: 148px;
+    object-fit: contain;
+    position: absolute;
+    bottom: 60px;
+    left: 137px;
+    -webkit-transform:rotate(18deg);
+    -moz-transform: rotate(18deg);
+    -ms-transform: rotate(18deg);
+    -o-transform: rotate(18deg);
+    transform: rotate(18deg);
+}
```

Comment:
Good use of `transform: rotate();` here this section is stating to look good.
You dont need all the vendor prefixes here any more as all the major browsers now support transform without a prefix